iac/stacks/api.py

```diff
@@ -279,6 +279,7 @@ def __init__(
             image=ecs.ContainerImage.from_docker_image_asset(image_asset),
             cpu=config.api.container_cpu,
             memory_limit_mib=config.api.container_memory,
+            security_groups=[container_security_group],
```

Comment:
Using the same strategy here as the other containers... Using an existing SG that already has DB access.